site/content/kapp-controller/docs/latest/packaging-tutorial.md

```diff
@@ -300,7 +300,8 @@ spec:
 EOF
 ```
 
-This Package contains some metadata fields specific to the verison, such as releaseNotes and a valuesSchema. The valuesSchema shows what configurable properties exist for the version. This will help when users want to install this package and want to know what can be configured.
+This Package contains some metadata fields specific to the version, such as releaseNotes and a valuesSchema. The valuesSchema shows what configurable properties exist for the version. This will help when users want to install this package and want to know what can be configured. Note that if you are using `ytt` for templating, you can [generate this value in OpenAPI v3 format from your ytt schema](faq.md#how-can-i-generate-the-valuesschema-from-my-ytt-schema).
```

Comment:
so i know this is the absolute worst,  but this file is generated from a different file.  the instructions (~ish) are on line 5 of this file https://github.com/vmware-tanzu/carvel/pull/301/files#diff-9daa3d29d76f56a42e1aa4f3d7058c710b105e761ad2c3477ac34def629659a5R5

Reply:
is it worth fully updating / inlining some of those steps into this tutorial?  happy to pair on it real quick,  but honestly i was thinking that my next step, if i wanted to make sure i actually understand what's going,  my next step would be to add a step to this tutorial where we generate the open api schema for the package ...

Reply:
I'm not tracking 100%... but I'm getting the idea that you wanted to hoist the instructions we are currently proposing go in the FAQ to be first-class/inlined instructions in the tutorial, itself.

Inlined instructions in the tutorial was my original goal, but saw it was gonna be some surgery to get there and opted to take one step back and make it more of a footnote.

I'm down with helping to get `ytt` to be more "suggested" — I think it's a better experience for folks (of course I would).